### PR TITLE
Travis integration

### LINKED ISSUE #28: Add TravisCI integration

### PR DESCRIPTION
**Pull request by mic4ael**
closes #28

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+install:
+  - pip install -r requirements.dev.txt
+  - pip install -e .
+  - npm install
+  - cd newdle/client
+  - npm install
+  - cd -
+before_script:
+  - psql -c 'CREATE DATABASE newdle_tests;' -U postgres
+  - cp newdle/newdle.cfg.example newdle/newdle.cfg
```
[the remark this answers is not in the document]
Comment (ThiefMaster) [reply .travis.yml]: also, there should not be any need for a config file to run the tests! if there is, that's a bug

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+script:
+  - pytest
+  - cd newdle/client;npm run test
```
Comment (ThiefMaster) [inline .travis.yml]: `make test` now runs both, so you can just use that here

**Comment by mvidalgarcia** [comment]
> closes #33

#28 **

**Comment by mic4ael** [comment]
> > closes #33
> 
> #28 **

Oh, yep. Thanks! ;)

**Pull request merged**